src/test/java/reposense/model/RepoLocationTest.java

```diff
@@ -0,0 +1,63 @@
+package reposense.model;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+
+import org.junit.Test;
+
+import reposense.parser.InvalidLocationException;
+import reposense.util.AssertUtil;
+
+public class RepoLocationTest {
+
+    @Test
+    public void repoLocation_parseGitUrl_success() throws Exception {
+        String gitUrl = "https://github.com/reposense/RepoSense.git";
+        RepoLocation repoLocation = new RepoLocation(gitUrl);
+        assertEquals(gitUrl, repoLocation.toString());
+        assertEquals("RepoSense", repoLocation.getRepoName());
```

Comment:
should we still assert the repo name and organization when u already asserted the `repoLocation.toString()`? Since it consists both repo name and organization

Reply:
Yes, we must still assert the repo name and organisation.
The objective of the test is to check whether the repo name, organisation and location are correctly parsed from the raw string given to the RepoLocation constructor.

Example: 
If I call `new RepoLocation("https://github.com/nus-oss/reposense.git#release")`
then the location, repo name and organisation must be as follows:
- location: `https://github.com/reposense/reposense.git`
- repo name : `reposense`
- organisation: `nus-oss`

Reply:
my bad, didn't look through it clearly